components/result-page.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/boxicons@latest/css/boxicons.min.css" />
+    <link rel="stylesheet" href="https://algernui.netlify.app/components/main.css" />
+
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/question-page.css">
+    <link rel="stylesheet" href="../styles/result-page.css">
+
+    <title>Algern Quiz</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <a href="/">
+                <h2 class="logo">Algern Quiz</h2>
+            </a>
+            <a href="#">
+                <i class='bx bxs-user icon'></i>
+            </a>
+        </nav>
+    </header>
+
+    <main>
+        <h2 class="text-center">Results: 5/10</h2>
+        <section class="question-container">
+            <img class="responsive-image question-img" src="../assets/holding-al-pacino-best-and-worst-film-moments.jpg"
+                alt="al pacino">
+            <article class="mcq-container">
+                <h2 class="m-0">Who is this actor?</h2>
+                <h3 class="m-0">Points: 5</h3>
+                <div class="mcq">
```

Comment:
I think instead of MCQ you can give some other name like choices or options or something else more suitable.

Reply:
I will certainly make the required changes, thank you @foolhardy21.